### PR TITLE
Ignore warnings

### PR DESCRIPTION
**Pull request by gpaul**
This PR makes parsing of expected errors more robust against warnings.

### CODE REVIEW

File: pkg/lvm/lvm.go [hunk 1]
```diff
@@ -660,6 +664,27 @@ func ignoreWarnings(str string) string {
 		if strings.HasPrefix(line, "WARNING") {
 			continue
 		}
+		// Ignore warnings of the kind:
```
Comment (jdef) [inline pkg/lvm/lvm.go]: are we bumping LVM so that we can detect this particular error message?

Comment (gpaul) [reply pkg/lvm/lvm.go]: I thought so, but no it appears unrelated.

I'm bumping lvm2 as i'm bumping it in dcos-enterprise, really just to get the latest fixes.

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ The following command-line utilties must be present in the `PATH`:
 * the filesystem listed as `-default-fs` (defaults to: `xfs`)
 
 This plugin's tests are run in a centos 7.3.1611 container with
-lvm2-2.02.171-8.el7 installed. It should work with newer versions of lvm2 that
+lvm2-2.02.177-8.el7 installed. It should work with newer versions of lvm2 that
```
Comment (jdef) [inline README.md]: if this package exists, then why build from source?

Comment (gpaul) [reply README.md]: The version 2.02.177 (latest) does not exist as a package, only the older one. I'd like to bring the version in the Dockerfile up to date with the latest LVM2 release.

**Comment by gpaul** [comment]
@jdef I've left the package as is, only modifying the parsing of warnings.

PTAL

**Comment by gpaul** [comment]
I've tested "volume create", too.

File: pkg/lvm/lvm.go (hunk 1 above)
Comment (jdef) [reply pkg/lvm/lvm.go]: there are a couple of MESOS tickets that could be related to this:
* https://issues.apache.org/jira/browse/MESOS-4065
* https://issues.apache.org/jira/browse/MESOS-8428
* https://issues.apache.org/jira/browse/MESOS-8128

**Comment by jdef** [comment]
why the last two commits (build/install from source)?

**Comment by gpaul** [comment]
> why the last two commits (build/install from source)?

There are a couple of reasons:
1. This way we test the plugin against the latest lvm2 release.
2. As we don't enable lvmetad we don't need to `sed` the `/etc/lvm/lvm.conf` file, which is a cool benefit.
3. The plugin runs in production in DC/OS Enterprise where we build this version of LVM2 from source. This gives me higher confidence when, in the README, we say that the plugin is tested with the given version of LVM2.

What do you think?

**Comment by jdef** [comment]
> What do you think?

Sounds reasonable to me

**Pull request merged**